backend/functions/views.py

```diff
@@ -2,179 +2,174 @@
-                translations_output = get_batch_translations_using_google_translate(
-                    sentence_list=batch_of_input_sentences,
-                    target_language=output_language,
-                    checks_for_particular_languages=checks_for_particular_languages,
-                )
-
-                # Check if translation output returned a list or a string
-                if isinstance(translations_output, str):
-                    return {"error": translations_output}
-                else:
-                    translated_sentences = translations_output
-
-            else:
-                return {"error": "Invalid API type. Allowed - [indic-trans, google]"}
-
-            # Check if the translated sentences are equal to the input sentences
-            if len(translated_sentences) != len(batch_of_input_sentences):
-                return {
-                    "error": "The number of translated sentences does not match the number of input sentences."
-                }
-
-            # Iterate through the dataframe
-            for index, row in input_sentences_df[i : i + batch_size].iterrows():
-
-                # Get the values for the TranslationPair model
-                sentence_text_id = row["sentence_text_id"]
-                input_sentence = row["corrected_text"]
-                input_language = row["input_language"]
-                translated_sentence = translated_sentences[index - i]
-                metadata = row["metadata"]
-                context = row["context"]
-
-                # Get the sentencetext model object by ID
-                sentence_text_object = dataset_models.SentenceText.objects.get(
-                    id=sentence_text_id
-                )
-
-                # Create and save a TranslationPair object
-                translation_pair_obj = dataset_models.TranslationPair(
-                    parent_data=sentence_text_object,
-                    instance_id=output_dataset_instance,
-                    input_language=input_language,
-                    output_language=output_language,
-                    input_text=input_sentence,
-                    machine_translation=translated_sentence,
-                    context=context,
-                    metadata_json=metadata,
-                )
-
-                # Append the object to TranslationPair list for bulk create
-                translation_pair_objects.append(translation_pair_obj)
-
-    # Bulk create the TranslationPair objects
-    multi_inheritance_table_bulk_insert(translation_pair_objects)
-
-    return "Success"
+# def sentence_text_translate_and_save_translation_pairs(
+#     languages,
+#     input_dataset_instance_id,
+#     output_dataset_instance_id,
+#     batch_size,
+#     api_type="indic-trans",
+#     checks_for_particular_languages=False,
+# ):
+#     """Function to translate SentenceTexts and to save the TranslationPairs in the database.
```

Comment:
Once the PR is tested, please remove all these comments before merging.

Reply:
Yes yes, I will remove them now.